File: challenges/codeeval/easy/174/edison_1520.cs
Builder result = new StringBuilder();
        for (Character theChar : fileLine.toCharArray()) {
            String slang = null;
            if (punctuacion.contains(theChar)) {
                if (Remplace = !Remplace) {
                    //slang = ObtenerSlang();
                    slang = slangs[Indice];
                    if (++Indice == slangs.length) {
                        Indice = 0;
                    }
                }
            }
            result.append(slang == null ? theChar : slang);
        }
        System.out.println(result.toString());
    }
    private static void Cambie(String[] args) throws Throwable {
    BufferedReader reader = new BufferedReader(new FileReader(new File(args[0])));
    String fileLine;
    while ((fileLine = reader.readLine()) != null) {
        if (!fileLine.isEmpty()) {
            slangFlavor(fileLine);
        }
    }
    }
}

File: challenges/codeeval/easy/173/edison_1520.java
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;

public class Main{
    public static void main(String[] args) throws Throwable {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String str;
        while ((str = buffer.readLine()) != null){
            char lastChar = str.charAt(0);
            lastChar++;
            StringBuilder result = new StringBuilder();
            for (char theChar : str.toCharArray()) {
            if (theChar != lastChar) {
                lastChar = theChar;
                result.append(theChar);
            }
        }
        System.out.println(result.toString());
        }
    }
}
